scout/server/blueprints/variant/utils.py

```diff
@@ -333,6 +333,14 @@ def predictions(genes):
     return data
 
 
+def set_colorsdb_freq(variant_obj: dict, freqs: dict) -> None:
```

Comment:
Don't know the code well enough, is this it's own function because colorsdb will only exist in long-read samples?

Reply:
No, it's more like because there is a set of conditions [before it is called](https://github.com/Clinical-Genomics/scout/blob/cc3bf24ab84d2f738c5205836c1783d74b46633f/scout/server/blueprints/variant/utils.py#L350), like:

if snvs:
elif svs:
elif mei:

And in these conditions are created the dictionaries. Sometimes different variant categories gave the same keys so there are repetitions, sometimes the keys are specific to the variant type.

That needs refactoring I think. Instead of having to set these dictionaries like that I'd like to introduce a call to a specific function like I'm doing here. It would avoid a lot of repetitions. Like in this case I would have to add the same key value both in the "if snvs" and "if svs".

Reply:
Okay, thanks for clarifying :)

Reply:
I'll just refactor in this PR, so colorsdb frequencies don't have to be weird compared with the other frequencies